docs/modules/ROOT/workflows/MQ_portcreate.puml

```diff
@@ -7,13 +7,23 @@ autonumber
 
 user -> PM: Create a port
 PM <-> SM_MACM_IPM_RM: PortInfo update
+<<<<<<< HEAD
```

Comment:
Line 10 is a merge conflict. Please review "<<<<<<<< HEAD"

Reply:
I have fixed MQ_portcreate.puml